components/mpas-seaice/bld/build-namelist-section

```diff
@@ -28,7 +28,7 @@ add_default($nl, 'config_full_abort_write');
 # Namelist group: decomposition #
 #################################
 
-add_default($nl, 'config_block_decomp_file_prefix', 'val'=>"'${DIN_LOC_ROOT}/ice/mpas-cice/${ICE_GRID}/mpas-cice.graph.info.${date_stamp}.part.'");
+add_default($nl, 'config_block_decomp_file_prefix', 'val'=>"'${DIN_LOC_ROOT}/ice/mpas-seaice/${ICE_GRID}/mpas-seaice.graph.info.${date_stamp}.part.'");
```

Comment:
We'll have to keep the old graph files with "cice" in the name.  Or make links for each individual one to the the new name.

Reply:
Couldn't we override this default for all meshes that have `mpas-cice` in their graph file names?  But I agree that there's work still to be done.

Reply:
I think we already do override this value for all meshes we support:
https://github.com/E3SM-Project/E3SM/blob/614358918ce3b79949bfc281ad8c8848661799e6/components/mpas-seaice/cime_config/buildnml#L75-L264

I must admit, though, that I don't understand what `build-namelist-section` does as opposed to `build-namelist`.

@jonbob, maybe you can give me some background either here or on slack?

Reply:
Right, we wouldn't rename any of the existing partition files.  They remain as `mpas-cice` in the list above form `buildnml`.  I just want to change the default name to be `mpas-seaice.graph.info.` for consistency. As far as I can tell, this default never gets used since all meshes (now and in the past) explicitly specify this prefix.

Reply:
@rljacob, I discussed this with @jonbob and this file isn't used directly for anything.  It is the automated output of a process used to update `build-namelist`, which is then manually combined with some customization (I as will be painfully clear, I'm fuzzy on the details) to get the final `build-namelist`.  So long story short, it doesn't matter what value is here from my understanding.